test/seldon/setup/manifests/model/seldon-model-ingress.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
this should be removed

Reply:
yaml file removed along with the line in script that references it.